src/command_modules/azure-cli-cognitiveservices/setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
```

Comment:
The setup.py should have the following:
```
try:
    from azure_bdist_wheel import cmdclass
except ImportError:
    from distutils import log as logger
    logger.warn("Wheel is not available, disabling bdist_wheel hook")
    cmdclass = {}
```

Also, it should have `cmdclass=cmdclass` in setup(...). See other modules for examples.

cc: @johanste